docs/developer/base/basics.md

```diff
@@ -1,3 +1,165 @@
 # Basics
 
 -----
+
+The AgentCheck base class in base.py contains the logic that all Checks inherit.
```

Comment:
`Check` is a common noun and should be lowercase.

Reply:
For our dev docs we keep `Check` capitalized to help us refer to them